src/modules/table.js

```diff
@@ -1060,7 +1061,11 @@ layui.define(['lay', 'laytpl', 'laypage', 'form', 'util'], function(exports){
   Class.prototype.resize = function(){
     var that = this;
 
-    if (!that.layMain) return;
+    var tableElemIsConnected = that.layMain && ('isConnected' in that.layMain[0]
+      ? that.layMain[0].isConnected 
+      : $.contains(document.body, that.layMain[0]));
+
+    if(!tableElemIsConnected) return;
```

Comment:
这里其实不用 `isConnected` 来判断，直接用 `$.contains` 就行啦，它内部本身也做了原生 `Node.contains()` 的兼容性处理。我没有注意到 jQuery 还有这个方法 😅
```suggestion
    if(!$.contains(document.body, that.layMain[0])) return;
```

Reply:
如果考虑到性能差异，上述 Suggestion 可忽略。

Reply:
是的，这个方法可能会高频率执行，Node.isConnected 性能更好。